### PR TITLE
vs-eng-Cleanup-on-the-ambience-manager-LV-420

### PR DESCRIPTION
**Pull request by RyanASwanson**
https://bradleycapstone.atlassian.net/jira/software/projects/LV/boards/32/backlog?assignee=6318bb796856bdd60aa045de&selectedIssue=LV-420

Adjusted the AmbienceManager and FmodAmbienceEvents to be more in line with the other managers. I don't have start in any other managers, reason being is that the core manager for both the universal manager and gameplay manager call SetUpMainManager on all their associated managers. Also adjusted the FmodAmbienceManager to set up the instance similarly to the managers.

There is an error with the interactables system. Ignore that it will be fixed elsewhere

Also this task is number 420. That's the weed number. I have the humor of a child

### CODE REVIEW

**Review by astapay — APPROVED** [review]
Looks good

File: Assets/General/Audio/FMODScripts/FmodAmbienceEvents.cs
Comment (ASLH68) [inline Assets/General/Audio/FMODScripts/FmodAmbienceEvents.cs:1]: Add self as contributor

File: Assets/General/Scripts/Managers/UniversalManagers/MainUniversalManagers/AmbienceManager.cs
Comment (ASLH68) [inline Assets/General/Scripts/Managers/UniversalManagers/MainUniversalManagers/AmbienceManager.cs:1]: Add self as contributor

Comment (ASLH68) [reply Assets/General/Scripts/Managers/UniversalManagers/MainUniversalManagers/AmbienceManager.cs:1]: Favor: Can you update the File Name in the header for me? :3 <3

**Review by ASLH68 — APPROVED** [review]

**Review by AlecatEnjoyer — APPROVED** [review]
I approved

**Pull request merged**